lib/utils/path.js

```diff
@@ -0,0 +1,28 @@
+var p = require('path');
+
+function isRelative(path) {
+  return path[0] === '.';
+}
+
+function makeRelative(from, to) {
+  var path = p.relative(p.dirname(from), to);
+  if (!isRelative(path)) {
+    path = './' + path;
+  }
+  return path;
+}
+
+function makeAbsolute(base, path) {
+  return p.resolve(p.dirname('/' + base), path).substring(1);
+}
+
+function startsWith(base, search) {
```

Comment:
I think we can use `String.prototype.startsWith` now (it is available in Node 4 and higher)